_posts/0000-01-02-Kathyeb1773.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
Mila Kitty is the Cutest!
![198485206_157644979683251_3885529846755976028_n](https://user-images.githubusercontent.com/85973518/122145870-a86bad80-ce1b-11eb-922b-c258fe7a5211.jpg)